app/views/users/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<!--<form accept-charset="UTF-8" action="/users" method="post">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    <label for="username">User:</label>
+    <input id="username" type="text" name="user[username]"></input>
+    <br/>
+    <label for="email">email:</label>
+    <input id="email" type="text" name="user[email]"></input>
+    <br/>
+    <label for="password">password:</label>
+    <input id="password" type="password" name="user[password]"></input>
+    <input name="commit" type="submit" value="Create" /><br/>
+</form>
+
+
+<%= form_tag users_path do %>
+
+  <%= label_tag 'User:' %>
+  <%= text_field_tag :'user[username]' %>
+  <br /><br />
+
+  <%= label_tag 'Email:' %>
+  <%= text_field_tag :'user[email]' %>
+  <br /><br />
+
+  <%= label_tag 'Password:' %>
+  <%= password_field_tag :'user[password]' %>
+  <br /><br />
+
+  <%= submit_tag "Create" %>
+<% end %>
+-->
+
+<%= form_for @user, url: {action: "create"} do |u| %>
```

Comment:
Looks clean like this; however, aren't we missing the label tags? Since this project is about learning to work with forms, let's add them.

[`REQUIRED`]